src/test/java/org/springframework/data/jpa/repository/UserRepositoryStoredProcedureTests.java

```diff
@@ -103,4 +142,37 @@ public void plainJpa21_entityAnnotatedCustomNamedProcedurePlus1IO() {
 
 		assertThat(proc.getOutputParameterValue("res"), is((Object) 2));
 	}
+
+	@Test // DATAJPA-707
+	@Ignore
```

Comment:
Because the other tests in this section are ignored too. It has been a few months and I don't remember why, but I left it Ignored to match the pre-existing tests